packages/cli/src/render.tsx

```diff
@@ -28,20 +28,33 @@ export const render = async () => {
 	loadConfigFile(configFileName);
 	parseCommandLine();
 	const parallelism = Internals.getConcurrency();
-	const renderMode = Internals.getFormat();
+	const renderMode = Internals.getOutputFormat();
 	const outputFile = getOutputFilename();
 	const overwrite = Internals.getShouldOverwrite();
 	const userProps = getUserProps();
 	const quality = Internals.getQuality();
 
 	const absoluteOutputFile = path.resolve(process.cwd(), outputFile);
+	const absoluteOutputFileWithExtension = (): string => {
+		if (renderMode === 'mp4') {
+			return absoluteOutputFile + '.mp4';
+		}
+		if (renderMode === 'webm-v8' || renderMode === 'webm-v9') {
+			return absoluteOutputFile + '.webm';
+		}
+		return absoluteOutputFile;
+	};
```

Comment:
The console message 'Your video is ready! -> {output location}' doesn't seem to contain the .webm extension.

Reply:
Just fixed this.